src/averageAges.js

```diff
@@ -15,11 +15,18 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let sumOfAges = 0;
+  let countOfMens = 0;
+
+  people.forEach(e => {
```

Comment:
avoid using loop and forEach

Reply:
I used one one loop. I think it's impossible to solve this task without loop.

Reply:
for example, you can use filter to get men. then map men to get ages